x/blob/keeper/params.go

```diff
@@ -21,12 +22,18 @@ func (k Keeper) SetParams(ctx sdk.Context, params types.Params) {
 
 // MinSquareSize returns the MinSquareSize param
 func (k Keeper) MinSquareSize(ctx sdk.Context) (res uint32) {
+	if ctx.BlockHeader().Height < 1 {
+		return appconsts.DefaultMinSquareSize
+	}
```

Comment:
[non blocking question] sorry I'm not familiar with defining genesis params. Is there a way to define the DefaultMinSquareSize param at chain genesis time so that we don't need this conditional?

If it isn't possible, why is a similar conditional not needed for the `GasPerBlobByte` param later in this file?

Reply:
I think it would be good to document this in the code

Reply:
Is it possible for someone to include a payForBlob in the first block of the chain? It seems unlikely but if yes, would `GasPerBlobByte` also need to be initialized?

Reply:
that's a good point, I don't think so, as the params are used in many places, but I'm not sure of what is done to make that the case! perhaps we can do that process a bit earlier, so that it works for processproposal, which would almost certainly be a better solution

Reply:
> Apparently the genesis params aren't set until the first block is committed

This seems odd. You should be able to set params in genesis that can be used for the first block.